test/itkScancoImageIOTest2.cxx

```diff
@@ -101,6 +101,30 @@ itkScancoImageIOTest2(int argc, char * argv[])
   ITK_TEST_EXPECT_TRUE(itk::Math::FloatAlmostEqual(scancoIO->GetMuScaling(), 4096.0, 6, 1e-3));
   itk::ExposeMetaData<double>(metaData, "MuScaling", doubleMeta);
   ITK_TEST_EXPECT_TRUE(itk::Math::FloatAlmostEqual(doubleMeta, 4096.0, 6, 1e-3));
+  std::cout << "MuWater: \t\t" << scancoIO->GetMuWater() << std::endl;
+  ITK_TEST_EXPECT_TRUE(itk::Math::FloatAlmostEqual(scancoIO->GetMuWater(), 0.7033, 6, 1e-3));
```

Comment:
Thanks for giving this a try @mkuczyns. 

Following @dzenanz 's comment, the right way to do this is the following: as all these parameter values are tied to the test image you are using, they should be passed as arguments to the test via specifying them in the `CMakeLists.txt`, e.g.

```
itk_add_test(NAME myTestName
      COMMAND ITKTestingDriverForModule
myTestMainMethodName DATA{Input/myInputImage.ISQ} 0.859375 0.85939 1.60016 0 1 0 0 1)
```

You will then need to adapt the test argument checking message, and parse all N parameters. Then, in order to test all values you would do something like:
```
auto muWater = std::stod(argv[2]);
scancoIO->SetMuWater(muWater):
ITK_TEST_SET_GET_VALUE(muWater, scancoIO->GetMuWater());
```

if the parameter index of `muWater` is 2, for example.

I know the previously existing tests were done the same way as you are doing; maybe you could take some time to refactor those as well, as e.g. 
https://github.com/KitwareMedical/ITKIOScanco/pull/64/files#diff-886d526d34a87d603536d350c146d93cd82880a1381276327dd3f2bfa18f757bR68 calling the getter method once to print the value and another time for the comparison is not efficient.

We usually try to avoid printing values unless necessary. It helps when debugging, but once we have verified that things work, we prefer the getter values to be exercised quantitatively with the test macros (e.g. `ITK_TEST_SET_GET_VALUE`) as shown above.

It is long, I know, but it is the only current way to:

1. Allow other input image and parameter values to be easily tested.
2. Make the test quantitative, so that whenever the body/implementation of the file changes, this is detected in the test, potentially failing if the new implementation is not correct.


You can read about testing in section "C.23 Writing Tests" of the ITK SW Guide: https://itk.org/ItkSoftwareGuide.pdf

Reply:
This is a lot of work. I think these tests were created with specific images in mind. While what Jon suggests would be better, it would also make sense to hardcode those image's file names in the tests. This contribution is good as-is (aside from minor clang-format issue).

Reply:
An additional note: all epsilon values in passed to the `FloatAlmostEqual` could be put into a constant for the sake of consistency/re-use.